src/Providers/Anthropic/Maps/MessageMap.php

```diff
@@ -86,10 +91,16 @@ protected static function mapUserMessage(UserMessage $message): array
             ];
         }, $message->images());
 
+        $cacheType = data_get($message->providerMeta(Provider::Anthropic), 'cacheType', null);
+
         return [
             'role' => 'user',
             'content' => [
-                ['type' => 'text', 'text' => $message->text()],
+                array_filter([
+                    'type' => 'text',
+                    'text' => $message->text(),
+                    'cache_control' => $cacheType ? ['type' => $cacheType instanceof UnitEnum ? $cacheType->name : $cacheType] : null,
```

Comment:
Instead of setting cache_control as null. can we not append the `cache_control` key only when caching is enabled.

Reply:
Thanks for reviewing! Array filter removes the null value, following the pattern in e.g. [OpenAI ToolMap](https://github.com/echolabsdev/prism/blob/main/src/Providers/OpenAI/Maps/ToolMap.php).

Reply:
yep make sense. I missed it.